packages/web-components/src/components/ic-dialog/ic-dialog.tsx

```diff
@@ -514,12 +514,11 @@ export class Dialog {
               <ic-button
                 class="close-icon"
                 variant="icon"
+                innerHTML={closeIcon}
                 aria-label={dismissLabel}
                 onClick={this.closeIconClick}
                 data-gets-focus={destructive || !buttons ? "" : null}
-              >
-                <span class="close-icon-svg" innerHTML={closeIcon} />
-              </ic-button>
```

Comment:
Has this been updated because there is a problem in Safari or a problem with the implementation? Would expect this to work as it's just an SVG in a span.

Reply:
What happens with it's just the SVG not wrapped in a span?

Reply:
This has been updated because of the Safari issue. In other places where close icons appear within icon buttons, we just use innerHTML to display them, this was the only place we used a span and I thought it'd be the most simple solution in order to get it in line with other components.

Reply:
Makes sense for consistency. Will approve.

It's worth creating another ticket to look into why the SVG didn't appear (within the span) in Safari in case an update is required within `ic-button`.

Reply:
I'll create a ticket now. It works within a span if you're slotting an icon into a regular button so potentially only an issue with the icon variant.